src/CMakeLists.txt

```diff
@@ -1,15 +1,21 @@
 
 set(CXXRT_SOURCES
     dynamic_cast.cc
-    exception.cc
     guard.cc
-    stdexcept.cc
     typeinfo.cc
     memory.cc
-    auxhelper.cc
```

Comment:
I don't think we should be removing this one.  It will leave us, for example, with no implementation of `__cxa_deleted_virtual`, which we still need even without exceptions.  I think the correct thing to do is either remove `__cxa_throw_bad_array_new_length` (it probably shouldn't be called when you don't have exceptions) from the no-exceptions build or make it call `abort()`.  That should make the file buildable without exceptions.

Reply:
I apologize, I missed these functions. I have changed to abort if the exception throwing functions are called.